### PR TITLE
[DE] Review `Global Moderation Team`

### PR DESCRIPTION
**Pull request by FreezyLemon**
- fixed some small grammar mistakes
- improved some incorrect or vague translations
- improved wording in some places
- added the note for translators above the tables

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/de.md [hunk 1]
```diff
@@ -11,65 +11,67 @@ tags:
...
-3. Habe ein grundlegendes Verständnis [der Regeln](/wiki/Rules) und wie diese angewendet werden. Es ist wichtig zu wissen, was akzeptabel innerhalb von osu! ist und was nicht.
-4. Sei geduldig. Das GMT fügt nicht spontan Mitglieder hinzu, die ein paar Tage geholfen haben. Es braucht Zeit für das GMT, um jemanden zu bewerten - seine Persönlichkeit, mit was er helfen kann, seine Aktivität, usw.
+1. Beweise dein gutes Verhalten durch höflichen und respektvollen Umgang mit anderen Nutzern. Es ist essenziell, deinen Account frei von Stummschaltungen oder Verstößen zu halten.
+2. Demonstriere deine Aktivität durch die Teilnahme am Chat und/oder in den Foren. [Melde Fehlverhalten](/wiki/Reporting_bad_behaviour) und Probleme, beantworte Fragen und hilf anderen mit Problemen.
+3. Habe ein gutes Verständnis [der Regeln](/wiki/Rules) und wie diese angewendet werden. Du solltest wissen, was in osu! akzeptabel ist und was nicht.
+4. Sei geduldig. Das GMT fügt nicht spontan Mitglieder hinzu, die ein paar Tage geholfen haben. Das GMT braucht Zeit, um jemanden zu bewerten – seine Persönlichkeit, mit was er helfen kann, seine Aktivität usw.
```
Comment (Deimosu) [inline wiki/People/Global_Moderation_Team/de.md:42]: mit was er helfen kann -> womit er helfen kann

**Review by Deimosu — APPROVED** [review]

File: wiki/People/Global_Moderation_Team/de.md [hunk 2]
```diff
@@ -11,65 +11,67 @@ tags:
...
+4. Mache nicht zu viel. Das Team sucht nicht nach übermäßig vielen, sondern nach regelmäßigen Beiträgen. Es ist bekannt, dass Kandidaten sonst ihre Geduld und ihr Engagement verlieren. Nur keine Hektik!
 
 ## Vorteile
 
-Globale Moderatoren erhalten die folgenden Vorzüge:
+Globale Moderatoren erhalten die folgenden Vorteile:
 
 - **[osu!supporter](/wiki/osu!supporter)**: Ein osu!supporter-Tag, das für die Dauer der Mitgliedschaft im Team gilt.
-- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Anzahl der Jahre anzeigt, die die Person im GMT war (beginnend mit 3).
+- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Amtszeit des Nutzers, beginnend mit 3 Jahren, anzeigt.
```
Comment (The-Last-Cookie) [inline wiki/People/Global_Moderation_Team/de.md:58]:
```suggestion
- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Beteiligung des Nutzers, beginnend mit 3 Jahren, anzeigt.
```
Maybe a bit too formal, though both are okay

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
Very nitpicky of me, awesome rework ^^

File: wiki/People/Global_Moderation_Team/de.md (hunk 2 above)
Comment (FreezyLemon) [reply wiki/People/Global_Moderation_Team/de.md:58]: `Beteiligung` kind of removes the context of time. I'm open to changing away from `Amtszeit`, but maybe include the time aspect?

Comment (The-Last-Cookie) [reply wiki/People/Global_Moderation_Team/de.md:58]: I can't think of a word that does express the time aspect as much as `Amtszeit` does, so let's just keep it for now.

**Review by The-Last-Cookie — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**